x-pack/legacy/plugins/canvas/canvas_plugin_src/functions/common/join_rows.ts

```diff
@@ -54,7 +54,7 @@ export function joinRows(): ExpressionFunction<'joinRows', Datatable, Arguments,
         aliases: ['sep', 'delimiter'],
         types: ['string'],
         help: argHelp.separator,
-        default: ',',
+        default: '","',
```

Comment:
I started double quoting all of the default values that are strings, so they would show up in the docs as "Default: `","`" instead of "Default: `,`" to make it clear that the value is a string value.